moulinette/__init__.py

```diff
@@ -62,7 +61,8 @@ def init(logging_config=None, **kwargs):
     configure_logging(logging_config)
 
     # Add library directory to python path
-    sys.path.insert(0, LIB_DIR)
+    # Note(decentral1se): must match moulinette.globals value
+    sys.path.insert(0, '/usr/lib/moulinette')
```

Comment:
Uh so here are we really sure we want to hardcode this value here ?

Reply:
OK, fair concern. I've taken another look and also used the `init_env` trick. It seems to work and the tests pass, so it looks like we can do this just fine. Please see 1854ab360bb32c17554f4394f63293685f42e700.